ci/jobs/macros.yaml

```diff
@@ -5,6 +5,15 @@
 # they're easy to find.
 ###
 
+# Inject the PYTHONUNBUFFERED env var into builds using a parameter
+- parameter:
+    name: python-unbuffered
+    parameters:
+      - string:
```

Comment:
I'll add a comment. PYTHONUNBUFFERED works if it has any value, so setting it to boolean and leaving it unchecked will set it to "false", which will turn PYTHONUNBUFFERED on. The empty string is the only way to turn it off.